go/test/endtoend/backup/vtbackup/backup_only_test.go

```diff
@@ -62,12 +62,20 @@ func TestTabletInitialBackup(t *testing.T) {
 	// Initialize the tablets
 	initTablets(t, false, false)
 
-	// Restore the Tablets
+	ver, err := getVTExecVersion("vttablet")
+	require.NoError(t, err)
+	if ver > 15 {
+		err := primary.VttabletProcess.CreateDB("testDB")
+		require.ErrorContains(t, err, "The MySQL server is running with the --super-read-only option so it cannot execute this statement")
```

Comment:
Better to use the mysql error code for this error as the text of an error message is not guaranteed to be stable, but the code is.